pyproject.toml

```diff
@@ -41,7 +41,7 @@ dependencies = [
     'pillow',
     'pyopengl==3.1.6',
```

Comment:
Here we wanted to test if this also works, when you bump pyopengl to a newer version. This version (3.1.6) was fixed at the same time as PyQt5, and therefore this may not be necessary any longer.

Reply:
PyOpenGL had to be fixed to this version because the newer version causes errors